Lib/wave.py

```diff
@@ -1,13 +1,14 @@
-"""Stuff to parse WAVE files.
+"""Module for parsing WAVE audio files.
 
 Usage.
 
 Reading WAVE files:
       f = wave.open(file, 'r')
-where file is either the name of a file or an open file pointer.
```

Comment:
Could you create a separate PR for these changes? We've removed duplicate docstrings from several modules (e.g. the venv module) to avoid future maintenance costs. I think we can also drop this giant docstring too.